### ISSUE TITLE
ghj

### ISSUE THREAD

**Issue by souravkole** [angular-slider/angularjs-slider#530]
### Steps to reproduce
1. Open the Demo Link provided below 
2. Scroll down at "Range slider with ticks and values at intermediate positions"
3. Showing step numbers in different position in IE browser(11). 

Demo: http://angular-slider.github.io/angularjs-slider/

### Expected behaviour
Step numbers should appear below the scale, as displayed in Chrome

![image](https://cloud.githubusercontent.com/assets/19322019/25999568/4aa6a2f4-3743-11e7-86df-155e6632babe.png)
### Actual behaviour
Step numbers are appearing on top of the scale, step number and selected step numbers are overlapping,
So it is not working properly in Internet Explorer

![image](https://cloud.githubusercontent.com/assets/19322019/25999406/9d85307c-3742-11e7-8d51-b0f01bdda5ed.png)

**Comment by ValentinH** [comment]
Indeed, this is a bug. Unfortunately, I don't have a Windows machine to try this on...
The issue might be to the following rule that could be broken on IE:
```css
.rzslider .rz-ticks-values-under .rz-tick-value {
    top: initial;
    bottom: -32px;
}
```

Could you try to inspect to css rules in IE11 to see if there is a warning?

**Comment by ValentinH** [comment]
Indeed, it must be due to `initial` not being supported in IE: https://www.w3schools.com/cssref/tryit.asp?filename=trycss_initial

**Comment by ValentinH** [comment]
The fix seems to replace `initial` by `auto`, but it requires more checks to ensure it's fine on the other browsers. Would you like to submit a pull request for this?

**Comment by anuragasaurus** [comment]
Can I work on this issue?

**Comment by ValentinH** [comment]
Sure ;)

**Comment by ValentinH** [comment]
Fix released under 6.1.2 ;)
